multi-agent/tests/conftest.py

```
@@ -557,46 +557,151 @@ def element_card():
 # =============================================================================
 
 def pytest_addoption(parser):
-    """Add custom command line options."""
+    """Add custom command line options.
+
+    Idempotent: pytest may load this conftest more than once (e.g. after a
+    manual pod copy / importlib), and re-adding the same flag raises ValueError.
+    """
+
+    def _addoption(*opts, **attrs):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_addoption`

Add return type annotation: `None`

(ANN202)